web/html/src/manager/admin/hub/peripherals.tsx

```diff
@@ -0,0 +1,54 @@
+import { hot } from "react-hot-loader/root";
+
+import * as React from "react";
+
+import { Button } from "components/buttons";
+import withPageWrapper from "components/general/with-page-wrapper";
+import { PeripheralsList } from "components/hub";
+import { SectionToolbar } from "components/section-toolbar/section-toolbar";
+import { HelpLink } from "components/utils";
+
+const IssPeripheral = () => {
+  const title = (
+    <div className="spacewalk-toolbar-h1">
```

Comment:
I think most places use `TopPanel` for this, so would be nice to use that here for consistency.